CurrentTimeApp/CurrentTimeApp.csproj

```diff
@@ -8,7 +8,7 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="7.0.0" />
+		<PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="7.0.1" />
```

Comment:
This dependency will be updated with https://github.com/jjliggett/CurrentTimeApp/pull/85